elasticsearch/1.10/README.md

```diff
@@ -0,0 +1,102 @@
+# How to use Elasticsearch on DC/OS
+
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) is a distributed, RESTful search and analytics engine.
+The [DC/OS Elasticsearch](https://github.com/mesos/elasticsearch) service is a Mesos framework that allows you to manage
+and use Elasticsearch in a flexible and scalable way.
+
+- Estimated time for completion: 5 minutes
+- Target audience: Data engineers and appops that want an search engine.
+- Scope: Install and use Elasticsearch.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Elasticsearch](#install-elasticsearch)
+- [Use Elasticsearch](#use-elasticsearch)
+- [Uninstall Elasticsearch](#uninstall-elasticsearch)
+
+## Prerequisites
+
+- A running DC/OS 1.10 cluster with 3 agents with each 1 CPU, 3 GB of RAM and 1 GB of disk space available.
+- [DC/OS CLI](https://dcos.io/docs/1.10/cli/install/) installed.
+
+## Install Elasticsearch
+
+To install Elasticsearch, do:
+
+```bash
+$ dcos package install elasticsearch
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#community-services
+This DC/OS Service is no longer maintained. See beta-elastic. The ElasticSearch DCOS Service implementation is alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.
+Continue installing? [yes/no] yes
+Installing Marathon app for package [elasticsearch] version [1.0.1-2]
+Elasticsearch is staging. Check Marathon and the ES management front-end for status.
+```
+
+After this, you should see the Elasticsearch service running via the `Services` tab of the DC/OS UI:
+
+![Elasticsearch DC/OS service](img/services.png)
```

Comment:
I think this is the 1.9 GUI, as this layout has changed in 1.10 - can you update the image with an image of the latest GUI ?